README.md

```diff
@@ -1,2 +1,164 @@
-# declarative-custom-elements
-Explainer draft for specifying an approach to Declarative Custom Elements
+In reference to https://github.com/WICG/aom/issues/192
+
+```html
+<fancy-label
+    for="input"
+    id="label"
+>
+  <!-- I've not seen talk of offering "for" functionality on other elements, so we likely need to leverage labelledby instead -->
```

Comment:
`for` would come from Customized Built-ins, which won't ship.

Can it also come from `ElementMixins` as it would be particularly nice for the relationship to bind with the click + focus forwarding, etc of a label, as well.